services/motion/motion.go

```diff
@@ -580,3 +583,25 @@ func PollHistoryUntilSuccessOrError(
 		time.Sleep(interval)
 	}
 }
+
+// MoveArm is a helper function to abstract away movement for general arms.
+func MoveArm(ctx context.Context, logger logging.Logger, a arm.Arm, dst spatialmath.Pose) error {
+	inputs, err := a.CurrentInputs(ctx)
+	if err != nil {
+		return err
+	}
+
+	model := a.ModelFrame()
+	_, err = model.Transform(inputs)
```

Comment:
I added some small optimizations into this function.  For one, we used to call `ComputePosition` here which effectively does nothing different from `model.Transform`.  I want to eventually delete the `ComputePosition` function entirely, but don't need to do that here